src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -82,6 +82,59 @@ std::string postUpdateContext
     return answer;
   }
 
+  //
+  // If upcr.contextElementResponseVector contains >= TWO responses and the second of them is a 'SccFound',
+  // then skip the first, the one that isn't 'SccFound'.
+  // Also, if the first of them is 472, then skip the second
+  //
+  // FIXME P6: Temporary hack for 'Entity found but Attribute not found':
+  //           See FIXME P6 'Temporary hack ...' 
+  //           in MongoCommonUpdate.cpp, function processContextElement
+  //
+
+
+  //
+  // Removing 'garbage' from the response vector (when just *one* incoming contextElement
+  //
+  
+  if (upcr.contextElementResponseVector.size() != parseDataP->upcr.res.contextElementVector.size())
+  {
+    for (unsigned int ix = 0; ix < upcr.contextElementResponseVector.size() - 1; ++ix)
+    {
+      //
+      // 2 cases:
+      //
+      // o 1. 472+302:  keep 302
+      //   Out[0]: (472) request parameter is invalid/not allowed
+      //   Out[1]: (302) Found
+      //
+      // o 2. 472+404: keep 472
```

Comment:
To keep track of this aparentely unneded "exception" I'd suggest to add the following comment regarding this case (or similar wording):

"This exception is not related with CPr forwarding functionality, but needed anyway. Otherwise 000_ngsi10_update/update_sequence.test steps 13 and 15 will break"

It doesn't hurt and may provide useful information in the future if we need to dig into the causes of this behaviour before removing the hack.